DOCS-5553 Adds Static Analysis Doc Set + Single Sourcing Logic for GitHub Actions and CircleCI Orb Documentation (#18343)
DataDog/documentation: content/en/continuous_integration/static_analysis/configuration.md

| `env`     | The environment you want your results tagged with. Datadog recommends using `ci` as the value for this input.                                                                           | False    | `none`          |
| `site`    | The [Datadog site][103] to send information to. Your Datadog site is {{< region-param key="dd_site" code="true" >}}.                                                                                 | False    | {{< region-param key="dd_site" code="true" >}}  |

Add the following to your CI pipeline:

```bash
# Install dependencies
npm install -g @datadog/datadog-ci
TEMP_DIR=$(mktemp -d) && curl -L -o $TEMP_DIR -O http://dtdg.co/latest-static-analyzer
unzip $TEMP_DIR/latest-static-analyzer

# Run Static Analysis (requires a pre-installed JVM)
$TEMP_DIR/cli-1.0-SNAPSHOT/bin/cli --directory . -t true -o results.sarif -f sarif

# Upload results
datadog-ci sarif upload results.sarif --service "$DD_SERVICE" --env "$DD_ENV"
```

[101]: /account_management/api-app-keys/#api-keys
[102]: /account_management/api-app-keys/#application-keys
[103]: /getting_started/site/

{{% /tab %}}
{{< /tabs >}}

> ### Import third-party Static Analysis results

Reviewer: ```suggestion
### Upload third-party static analysis results to Datadog
```

> Datadog supports ingesting Static Analysis results from third-party Static Analysis tools that can export results into the interoperable [Static Analysis Results Interchange Format (SARIF) Format][3]. 

Reviewer: ```suggestion
You can send results from third-party static analysis tools to Datadog, provided they are in the interoperable [Static Analysis Results Interchange Format (SARIF) Format][3]. 
```

To upload a SARIF report:

1. Ensure the [`DD_API_KEY` and `DD_APP_KEY` variables are defined][2].
2. Install the `datadog-ci` utility:
   
   ```bash
   npm install -g @datadog/datadog-ci
   ```

> 3. Run your third-party Static Analysis tool on your code which outputs results in the SARIF format.

Reviewer: ```suggestion
3. Run the third-party static analysis tool on your code and output the results in the SARIF format.
```

4. Upload the results to Datadog:

   ```bash

>    datadog-ci sarif upload $OUTPUT_LOCATION --service <datadog-service> --env <datadog-env>

Reviewer: ```suggestion
   datadog-ci sarif upload $OUTPUT_LOCATION --service <datadog-service> --env <datadog-env> --site <dd-site>
```
